code/modules/cargo/packs.dm

```diff
@@ -550,11 +550,14 @@
 
 /datum/supply_pack/security/armory/bulletarmor
 	name = "Bulletproof Armor Crate"
-	desc = "Contains three sets of bulletproof armor. Guaranteed to reduce a bullet's stopping power by over half. Requires Armory access to open."
-	cost = 1200
+	desc = "Contains three sets of bulletproof armors with their corresponding helmets. Guaranteed to reduce a bullet's stopping power by over half. Requires Armory access to open."
+	cost = 3000
 	contains = list(/obj/item/clothing/suit/armor/bulletproof,
 					/obj/item/clothing/suit/armor/bulletproof,
-					/obj/item/clothing/suit/armor/bulletproof)
+					/obj/item/clothing/suit/armor/bulletproof,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt)
```

Comment:
Needs an NSV tag

Reply:
added the tag in [98bb89c](https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)